css-layout-classes/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <title>css layout classes</title>
+</head>
+
+<body>
+  <div class="container">
+
+    <div class="row">
+      <div class="column-full">
+        <header><h1>Choose your character!</h1></header>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-third">
+        <img src="images/link.png" alt="Zelda, jk Link">
+        <aside aria-label="main">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>Super Smash Bros. Ultimate received universal acclaim from both critics and players, with some critics calling it the best installment in the series. It received praise for its large amount of content and fine-tuning of existing Smash gameplay elements, although its online mode was widely criticized.</p>
+        </aside>
+        <img src="images/yoshi.png" alt="Yoshi">
+      </div>
+    </div>
+
+    <article>
+      <div class="row">
+        <div class="column-one-fourth">
+          <img src="images/kirby.png" alt="Kirby">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/fox.png" alt="Fox">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/pikachu.png" alt="Yellow Rat">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/ness.png" alt="Ness">
+        </div>
+      </div>
+    </article>
+    <div class="column-one-third">
```

Comment:
there should be a row for each of the bottom two rows, which have two columns within. Try to update to match that idea.